### ISSUE TITLE
[Bug] If I launch a campaign without "Executor", I have an error

### ISSUE THREAD

**Issue by nicodeur** [jenkinsci/cerberus-testing-plugin#12]
Since version `1.6.3`, When I launch a campaign, I have this error :

```
java.lang.NullPointerException

	at org.cerberus.launchcampaign.executecampaign.ExecuteCampaignDto.<init>(ExecuteCampaignDto.java:79)

	at org.cerberus.jenkinsci.plugins.executecerberustest.ExecuteCerberusCampaign.perform(ExecuteCerberusCampaign.java:155)

	at org.jenkinsci.plugins.workflow.steps.CoreStep$Execution.run(CoreStep.java:80)

	at org.jenkinsci.plugins.workflow.steps.CoreStep$Execution.run(CoreStep.java:67)

	at org.jenkinsci.plugins.workflow.steps.SynchronousNonBlockingStepExecution.lambda$start$0(SynchronousNonBlockingStepExecution.java:47)

	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)

	at java.util.concurrent.FutureTask.run(FutureTask.java:266)

	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)

	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)

	at java.lang.Thread.run(Thread.java:748)
```

If I type an Cerberus `executor` on Jenkins Global Configuration or on the job cerberus configuration, it works fine

**Comment by nicodeur** [comment]
Effectively, solution it's to type an `executor` on Jenkins Global Configuration :

![image](https://user-images.githubusercontent.com/6707556/65884913-fa477800-e399-11e9-88ca-a57616940048.png)

We will do a fix with a default value `Jenkins`on this attribute

**Comment by nicodeur** [comment]
Fix on version `1.6.4`
